pkg/auth/authz_handler.go

```diff
@@ -118,6 +123,24 @@ func OrgIDFromContext(ctx context.Context) string {
 	return payload.Organization
 }
 
+// EmailFromContext returns email from the specified context
+func EmailFromContext(ctx context.Context) string {
```

Comment:
do we have a test for this? maybe need to check an empty string and valid string

Reply:
Done